docs/src/pages/docs/Miscellaneous/issue_codes.mdx

```diff
@@ -167,11 +168,18 @@ Either the database is spelled incorrectly or does not exist.
 
 Either the database was written incorrectly or it does not exist. Check that it was typed correctly.
 
-
 ## Issue 1016
 
 ```
 The schema was deleted or renamed in the database.
 ```
 
 The schema was either removed or renamed. Check that the schema is typed correctly and exists.
+
+## Issue 1017
+
+```
+We were unable to connect to your database. Please confirm that your service account has the Viewer and Job User roles on the project.
+```
+
+The user doesn't have the proper permissions to connect to the database
```

Comment:
The convention we've been using is:

```
    ```
    Short message describing the problem.
    ```

    Descriptive message with actionable items.
```

So in this case it would be better to reverse the 2 messages you have here.